backend/src/schemas/models/tool.py

```
@@ -0,0 +1,85 @@
+import json
+import sqlalchemy as sa
+from sqlalchemy import Column, String, DateTime, Text, ForeignKey, Boolean
+from sqlalchemy.sql import func
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.orm import relationship
+
+from src.services.db import get_db_base
+
+class Server(get_db_base()):
```

Pylint (3.3.7)

[refactor] Too many instance attributes (8/7)

(R0902)